src/queueserver.ts

```diff
@@ -291,13 +291,27 @@ export const submitPlan = async(submitPlan: ISubmitPlanObject): Promise<IPlanObj
             } else {
                 plan.kwargs[key] = Number(value[0])
             }
-        }
-      } else {
-        plan.kwargs[key] = value;
+	} else {
+		if (key == "md"){
+		plan.kwargs[key] = {"foo": "bar"};
```

Comment:
Looks like a leftover entry. Should it be something more realistic, or an empty dict?

Reply:
I think this can be removed, this was part of debugging, and I think it didn't solve our problem.